modules/display-books.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable max-classes-per-file */
+export class BooksLocalStorage {
+  static getBookStorage() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static addBookStorage(book) {
+    const books = BooksLocalStorage.getBookStorage();
```

Comment:
- [ ] Nice work. As part of the requirements, kindly change all your classes and methods to arrow functions

Reply:
Done, thanks for pointing it out